api/src/main/java/org/motechproject/nms/api/web/KilkariController.java

```diff
@@ -78,7 +76,7 @@ public InboxResponse getInboxDetails(@RequestParam(required = false) Long callin
     }
 
     private void validateSaveInboxCallDetailsContent(StringBuilder failureReasons,
-                                                     Set<InboxCallDetailsRequestCallData> content) {
+                                                     Set<CallDataRequest> content) {
```

Comment:
There isn't a CallData request per se, right? CallData object is just part of a list inside of CallDetailsRequest?